test/rauc-hawkbit-updater.t

```diff
@@ -61,7 +61,7 @@ test_expect_success "rauc-hawkbit-updater register and check (invalid auth token
 
 test_expect_success "rauc-hawkbit-updater register and check (no security tokens)" "
   cp $SHARNESS_TEST_DIRECTORY/test-config-no-security-tokens.conf . &&
-  echo 'Loading config file failed: Neither auth_token nor gateway_token is set in the config.' > expected_out &&
+  echo 'Loading config file failed: Neither a token nor client certificate are set!' > expected_out &&
```

Comment:
The bash tests are gone since #83. It would be great to have an actual test case for this with a reverse proxy. We already use nginx for some test cases (see test/conftest.py).